### DISCUSSION TITLE
Can't install packages

### DISCUSSION THREAD

**Discussion by have-no-clue-what-im-doing** [AvaloniaUI/Avalonia#8909] [Q&A]
Windows 10
Have SDK Version: 6.0.400 installed
Have the Avalonia extension installed
When running the 
``` 
dotnet new -i Avalonia.Templates 
```
It returns the error: Avalonia.Templates could not be installed, the package does not exist.
Not sure what to do. Are there any other dependencies / requirements that need to be installed first?

**Comment by timunie** [comment]
We have seen some people having issues with nuget package manager at all. Can you double-check if nuget works for you in Rider or Visual Studio?

**Reply by have-no-clue-what-im-doing** [reply]
Not working in VS either. I'm new to all of this so I'm not sure if I screwed up when installing everything.

**Reply by have-no-clue-what-im-doing** [reply]
Yeah, just tried it in a VM with just the SDK installed and it worked. Think I installed too much shit at once. VS, .NET, extension.

**Reply by have-no-clue-what-im-doing** [reply]
Figured it out. Uninstalled everything. Reinstalled .NET SDK. Deleted the Nuget config in  %AppData%\NuGet
